### ISSUE TITLE
jsx-no-target-blank false positive

### ISSUE THREAD

**Issue by jahirfiquitiva** [yannickcr/eslint-plugin-react#2549]
I have enabled `react/jsx-curly-brace-presence` but now the `react/jsx-no-target-blank` rules is considering the following snippet as error:

```javascript
<a className={'App-link'}
         href={'https://reactjs.org'}
         target={'_blank'}
         rel={'noopener noreferrer'}>Learn React</a>
```

What can I do to fix it so it doesn't consider it an error?

**Comment by ljharb** [comment]
Duplicate of #2508; will be solved by #2544.
